### ISSUE TITLE
Subdirectory path not working?

### ISSUE THREAD

**Issue by Klowner** [ofrogon/cleanBowerInstaller#14]
I'm fairly certain I'm doing this correctly, at least according to the documentation and examples.

bower.json:

``` json
{
  "name": "test-project",
  "dependencies": {
    "underscore": "*"
  },
  "cInstall": {
    "option": {
      "default": {
        "folder": "public"
      }
    },
    "source": {
      "underscore": {
        "*#/mySubdir": "underscore.js"
      }
    }
  }
}

```

I would expect to have `public/mySubdir/underscore.js` but I end up with `public/underscore.js` instead.

Thanks!

**Comment by ofrogon** [comment]
Hi @Klowner,

Nice catch! It seems like I broke this functionality quite long ago... And my [Real use case](https://github.com/ofrogon/cleanBowerInstaller/wiki/Examples#real-use-case-from-uctrl-website) example doesn't even work right now.

I will release the fix this week end.

**Comment by ofrogon** [comment]
Fix in version 0.3.3

Thank you for reporting bugs!
